crates/core/src/globals.rs

```diff
@@ -455,11 +455,16 @@ fn build_memory(context: &JSContextRef) -> anyhow::Result<JSValueRef> {
             if !ptr.is_number() {
                 bail!("Expected a pointer");
```

Comment:
while you're in here....

do you think it makes sense to change some of these messages where `pointer` is used, to instead match the JS function names which use `offset`? 

e.g.
```patch
- bail!("Expected a pointer");
+ bail!("Expected an offset");
```

Reply:
Good call! Done in the latest commit.